airbyte-integrations/bases/base-python/base_python/cdk/streams/core.py

```diff
@@ -102,12 +107,38 @@ def cursor_field(self) -> Union[str, List[str]]:
         return []
 
     @property
-    def source_defined_cursor(self) -> bool:
+    def cursor(self) -> bool:
```

Comment:
can either keep this as `source_defined_cursor`? this one makes sense because the user actually can input a cursor, and this just toggles that